readme.md

```diff
@@ -281,7 +281,93 @@ cat /data/new_zealand.polylines | docker run -i -v /data:/data pelias/interpolat
 
 ### running a build in the docker container
 
-the build scripts are configurable via environment variables, you will need to download your data before running the build command.
+The build scripts are configurable via a combination of environment variables and a `pelias-config` json file.
+You will need to download your data before running the build command.
+
+To make use of the `pelias-config` functionality, you'll need to create a new json file called `pelias.json` for example.
+The relevant parts of that new file should look as follows. To direct to download script to this file, 
+the `PELIAS_CONFIG` environment variable should be set. You can read more details on how to use the `pelias-config` module
+[here](https://github.com/pelias/config).
+
+```javascript
+{
+  "imports": {
+    "interpolation": {
+      "download": {
+        "tiger": {
+          "datapath": "/tmp/data/tiger/",
+          "states": [
+            {
+              "state_code": 41
+            }
```

Comment:
I was thinking this would allow us to add more properties for each state later or to allow the use of `state_abbreviation` or something else to indicate the state. I'm not very attached to this.